README.md

```diff
@@ -25,14 +25,18 @@ The **source code** is available on [GitHub](https://github.com/Fullscreen/yt-au
 ## Usage
 
 ```ruby
-Yt::Audit.has_info_cards?('rF711XAtrVg')
-# => true
-Yt::Audit.has_brand_anchoring?('rF711XAtrVg', 'Budweiser')
-# => true
-Yt::Audit.has_subscribe_annotations?('rF711XAtrVg')
-# => false
-Yt::Audit.has_link_to_own_channel?('rF711XAtrVg')
-# => false
-Yt::Audit.has_end_cards?('rF711XAtrVg')
-# => false
+audit = Yt::Audit.for('UCPCk_8dtVyR1lLHMBEILW4g')
+# => #<Yt::Audit:0x007f822ba18f38 @info_card_count=10, @brand_anchor_count=10, @subscribe_count=0, @association_count=0, @end_card_count=0, @videos_count=10>
+audit.videos_count
```

Comment:
If you use singular everywhere else (e.g. `info_card`), then you should probably make this singular too for coherence (`video_count`)